uamqp/error.py

```diff
@@ -7,10 +7,138 @@
 from enum import Enum
 from collections import namedtuple
 
-from .constants import PORT, FIELD
+from .constants import SECURE_PORT, FIELD
 from .types import AMQPTypes, FieldDefinition
 
 
+class ErrorCodes(bytes, Enum):
```

Comment:
Question - what's the difference between this an the `ErrorCondition` enum on line 142? Can we consolidate them?

Reply:
yes we can, the `ErrorCondition` is not used so we could remove it, as well as the Connection/Session/LinkCondition

Reply:
I think I prefer the name `ErrorCondition` to `ErrorCode`.
Also - we should keep the documentation.

Reply:
updated!